presto-orc/src/main/java/com/facebook/presto/orc/OrcRecordReader.java

```diff
@@ -81,6 +83,7 @@
     private long currentPosition;
     private long currentStripePosition;
     private int currentBatchSize;
+    private int nextBatchSize = INITIAL_BATCH_SIZE;
```

Comment:
How about we add a constructor argument for `INITIAL_BATCH_SIZE`, so we don't have to modify the raptor tests?